sdk/communication/azure-communication-identity/README.md

```diff
@@ -73,8 +73,11 @@ print("User created with id:" + user.properties['id'])
 
 Use the `get_token` method to issue or refresh a scoped access token for the user. \
 Pass in the user object as a parameter, and a list of `CommunicationTokenScope`. Scope options are:
-- `CHAT` (Chat)
-- `VOIP` (VoIP)
+- `chat` (Use this for full access to Chat APIs)
```

Comment:
Why are the scopes in lowercase? We’re referring to the enum values of CommunicationTokenScope, so shouldn’t they be in uppercase?